src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import duke.task.Task;
+
+import java.util.ArrayList;
```

Comment:
I believe java imports should be placed above any other imports you have. I'm not sure if it's explictly stated in the Java coding standard, but it will show up in checkstyle if you do decide to implement it later on.

Reply:
You're right, it caused a checkstyle error and has to precede the rest of the imports. Thanks!